hazelcast/proxy/vector_collection.py

```diff
@@ -80,6 +168,25 @@ def put_all(self, map: Dict[Any, Document]) -> Future[None]:
         return combine_futures(futures)
 
     def put_if_absent(self, key: Any, document: Document) -> Future[Optional[Document]]:
+        """Associates the specified key with the given Document if it is not
+        already associated.
+
+        Warning:
+            This method returns a clone of the previous Document, not the original
+            (identically equal) Document previously put into the VectorCollection.
+
+        Warning:
+            This method uses ``__hash__`` and ``__eq__`` methods of binary form
+            of the key, not the actual implementations of ``__hash__`` and
+            ``__eq__`` defined in key's class.
+
+        Args:
+            key: Key of the entry.
+            document: Document of the entry.
+
+        Returns:
```

Comment:
if it's absent, how it will return the old Document?

Reply:
https://github.com/hazelcast/hazelcast-python-client/pull/678/commits/75ff67f5ca7811e094e2a3603b04e59f51eeceab